wp-content/themes/core/assets/js/src/utils/dom/smooth-anchors.js

```diff
@@ -19,20 +19,40 @@ const handleAnchorClick = ( e ) => {
 		offset: -150,
 		duration: 300,
 		$target: $( target ),
+		afterScroll: function() {
+			if ( ! options.auto_focus ) {
+				return;
+			}
+
+			if ( ! target.getAttribute( 'tabindex' ) ) {
+				target.setAttribute( 'tabindex', '-1' );
```

Comment:
I just so happen to be digging into browser skip link issues so this matter is top of mind at the moment 😂

Fwiw, I was just reminded this utility only works on anchors that serve as an anchor hash link, so we don't need the tabindex functionality, and I'm going to commit the a11y tweaks I'm making to handle broken skip links, which I think will handle the focus bit.

Let's tag team this, I'll commit my work in this branch (still working on it, will have it wrapped today), you comment out and/or remove everything but the fixed callback, and then let's both test and see where we land on this? Thanks for noticing this and bringing it up!

Reply:
Sounds good, we can definitely tag team this.

I'm a little confused though why you wouldn't need the tabindexing. The smooth scroll's destination could be anything with an `id` so like Dave mentioned it could be a non-focusable element. I'm picturing an author linking in a nav menu item to a specific block on that page.

Either way, I'll wait for your commit and make tweaks to mine based on that.

Reply:
Scope this: https://github.com/moderntribe/square-one/pull/677/files#diff-3322c350466fd62a27abec4bb2f60e0638b414251d64b083108843ccf413c699R37

We only apply this overall utility to anchors that have an anchor hash href value : )

Reply:
Oh derp, ignore me, right, the thing it might reference may not be focusable. Haha, what I'm doing up for a11y will also address that too : )

Reply:
No problem at all! Great! Looking forward to seeing what you come up with.